packages/flutter/lib/src/widgets/icon.dart

```diff
@@ -88,33 +94,95 @@ class Icon extends StatelessWidget {
   ///
   /// Icons occupy a square with width and height equal to size.
   ///
-  /// Defaults to the current [IconTheme] size, if any. If there is no
-  /// [IconTheme], or it does not specify an explicit size, then it defaults to
-  /// 24.0.
+  /// Defaults to the nearest [IconTheme]'s [IconThemeData.size].
```

Comment:
Do we have a default for when the IconTheme doesn't provide a size?

Same question for all the new properties as well.

Reply:
There's a fallback, which is documented in IconThemeData.size (and similarly for all properties).

Reply:
Fair enough. We are somewhat inconsistent on where we document defaults (in the widget or the theme), but we can do a sweep for these later.

Reply:
I like the idea of documenting defaults only where they're set, in most cases that'll be the widget. In this case, the theme.